internal/deviations/deviations.go

```diff
@@ -596,3 +596,9 @@ func ISISCounterManualAddressDropFromAreasUnsupported(dut *ondatra.DUTDevice) bo
 func ISISCounterPartChangesUnsupported(dut *ondatra.DUTDevice) bool {
 	return lookupDUTDeviations(dut).GetIsisCounterPartChangesUnsupported()
 }
+
+// SkipTcpNegotiatedMssCheck returns true for devices that do not
+// support telemetry to check negotiated tcp mss value.
+func SkipTcpNegotiatedMssCheck(dut *ondatra.DUTDevice) bool {
```

Comment:
Fix the static analysis error here by renaming it to SkipTCPNegotiatedMssCheck

Reply:
updated.